plugins/recorded_future/komand_recorded_future/actions/download_hash_risk_list/action.py

```diff
@@ -16,8 +18,18 @@ def __init__(self):
         )
 
     def run(self, params={}):
-        query_params = {"format": "xml/stix/1.2", "gzip": "false"}
+        query_params = {"format": "xml/stix/1.2", "gzip": "true"}
         risk_list = AvailableInputs.HashRiskRuleMap.get(params.get(Input.LIST))
         if risk_list:
             query_params[Input.LIST] = risk_list
-        return {Output.RISK_LIST: self.connection.client.make_request(Endpoint.download_hash_risk_list(), query_params)}
+        response_content, response_dict = self.connection.client.make_request(
+            Endpoint.download_hash_risk_list(), query_params
+        )
+        data = {
+            Output.RISK_LIST: response_dict,
+            Output.RISK_LIST_GZIP: {
+                "filename": "risk_list.gzip",
+                "content": base64.b64encode(response_content).decode("utf-8"),
```

Comment:
Suggest moving `utf-8` to constants but its up to you 😄

Reply:
We only use this once so I'd rather keep it here as a string.